README.md

```diff
@@ -6,42 +6,69 @@
 
 # ssi-service
 
-## Introduction
+HTTP-API driven web service that wraps our [ssi-sdk](https://github.com/TBD54566975/ssi-sdk).
 
+## Introduction
 The Self Sovereign Identity Service (SSIS) facilitates all things relating to [DIDs](https://www.w3.org/TR/did-core/)
-and [Verifiable Credentials](https://www.w3.org/TR/vc-data-model) -- in a box! The service is a part of a larger
+and [Verifiable Credentials](https://www.w3.org/TR/vc-data-model) - in a box! The service is a part of a larger
 Decentralized Web Platform architecture which you can learn more about in our
-[collaboration repo](https://github.com/TBD54566975/collaboration). The SSI Service is a HTTP-API driven web service
-that wraps the [ssi-sdk](https://github.com/TBD54566975/ssi-sdk). The core functionality of the SSIS includes,
-but is not limited to: interacting with the standards around Verifiable Credentials, Credential Revocations, requesting
-Credentials, exchanging Credentials, data schemas for Credentials and other verifiable data, messaging using
-Decentralized Web Nodes, and usage of Decentralized Identifiers. Using these core standards, the SSIS enables robust
-functionality to facilitate all verifiable interactions such as creating, signing, issuing, curating, requesting,
-revoking, exchanging, validating, verifying credentials in varying degrees of complexity.
-
-For more information, see the [vision document](doc/VISION.md).
-
-![ssi-service](doc/ssi-service.png)
+[collaboration repo](https://github.com/TBD54566975/collaboration).
```

Comment:
can we change this link to point to the dev website?

Reply:
the ssi-service page on the dev site is just this README file...https://developer.tbd.website/projects/ssi-service/README

so i'd say either docs home page: https://developer.tbd.website/docs/intro or ssi-service docs page: https://developer.tbd.website/docs/apis/ssi-service

thoughts?

Reply:
docs home page

Reply:
+1 to repointing this somewhere on the Dev Site. 

I think we should do two things - 

1) Point this to the Web5 projects page (https://developer.tbd.website/projects/web5), as that's what it's referencing. 
2) We rev that Projects page to be more clear, have better calls to action, more concretely outline the components which make up Web5.

Offshoring to docs home doesn't make sense to me as the link target here.

Reply:
So the only thing i don't love about the projects page is it's a lot of words in a developers face and the first one DIDs links to w3c so you've already lost me in a way

Reply:
Yes, the Projects page absolutely needs a refresh. https://github.com/TBD54566975/developer.tbd.website/issues/326

My recommendation is to have ssi-service point to Web5 Projects page, and we make sure that one does what it needs to do on the Developer Site.

Reply:
So for now are you saying point to projects page or after we redo the projects page we will point to it?

Reply:
Totally your call so you can close out on this issue without blocking on more work.

Medium-term I'm seeing how important it is we evolve the Project/Component landing pages on the Dev Site. Your links here deserve to point somewhere great.